NodePropertyPalette/PropertyPaletteNodeViewModel.cs

```diff
@@ -50,6 +50,10 @@ public NodeBuiltInStatus IsBuiltIn
                     var mangledName = (NodeModel as DSFunctionBase).Controller.MangledName;
                     return KnownDynamoNodeNamespaces.Any(ns => mangledName.StartsWith(ns)) ? NodeBuiltInStatus.Yes : NodeBuiltInStatus.No;
                 }
+                if (NodeModel is Dynamo.Graph.Nodes.CustomNodes.Function)
+                {
+                    return NodeBuiltInStatus.No;
```

Comment:
@mmisol This is to fix the bug that custom nodes will be displayed as built-in nodes